src/ALObject.ts

```diff
@@ -68,36 +73,37 @@ export class ALObject {
             case 'pageextension':
             case 'tableextension':
             case 'enumextension': {
-                let patternObject = new RegExp(`(\\w+) +([0-9]+) +(${ObjectNamePattern}|${ObjectNameNoQuotesPattern}) +extends +(${ObjectNamePattern}|${ObjectNameNoQuotesPattern})\\s*(\\/\\/\\s*)?([0-9]+)?`);
+                let patternObject = new RegExp(`(\\w+) +([0-9]+) +(${ObjectNamePattern}|${ObjectNameNoQuotesPattern}) +extends +(${ObjectNamePattern}|${ObjectNameNoQuotesPattern})\\s*(\\/\\/\\s*)?([0-9]+)?(\\s*\\(([0-9]+)?\\))?`);
```

Comment:
´patternObject´ vs ´objectPattern´? On line 163 ´xliffIdTokenPattern´ is declared.

Reply:
I realize the var name is not part of new development. So maybe we should leave this for my future refactoring project?